public/lib/content-service.js

```diff
@@ -146,7 +145,7 @@ angular.module('wfContentService', ['wfHttpSessionService', 'wfVisibilityService
                  */
                 updateField(contentItem, field, data) {
 
-                    if (field === 'status' && !contentItem.composerId) {
+                    if (field === 'status' && contentItem.status === 'Stub') {
```

Comment:
@susiecoleman Nope, Stubs are created when you start content in "News List" this means it hasn't got a related composer article yet.